README.md

```
@@ -688,13 +692,140 @@ collection2.ReplaceOne((Predicate<dynamic>)(e => e.id == 11), dynamicUser);
 
 ### Decimal precision
 
-Large `decimal` values do not survive a round trip cleanly. Data is internally routed through `ExpandoObject`, which represents JSON numbers as `double`, so anything past roughly 15 significant digits becomes inexact. `decimal.MaxValue` is a worst case — it is written in scientific notation (`7.922816251426434E+28`) that cannot be parsed back as `decimal` at all, and reading the file raises `JsonReaderException`.
+Large `decimal` values do not survive a round trip cleanly. Data is internally routed through `ExpandoObject`, which represents JSON numbers as `double`, so anything past roughly 15 significant digits becomes inexact. `decimal.MaxValue` is a worst case — it is written in scientific notation (`7.922816251426434E+28`) that cannot be parsed back as `decimal` at all, and reading the file raises `System.Text.Json.JsonException` (in earlier Newtonsoft-based releases this surfaced as `Newtonsoft.Json.JsonReaderException`).
 
 If you need exact preservation of large or high-precision decimals, store them as strings. Pinned by `NumericEdgeCaseTests`.
 
-## C# Language Version
+## System.Text.Json vs Newtonsoft.Json
```

markdownlint-cli2 (0.22.1)

[warning] Heading style
Expected: setext; Actual: atx

(MD003, heading-style)